src/scripts/modules/media/body/content-baked.less

```diff
@@ -207,9 +207,12 @@ h1.example-title .text {
 .os-problem-container,
 .os-solution-container {
   display: inline;
-  > :first-child {
+  > :first-child:not(ul):not(ol) {
```

Comment:
`> :first-child:not(ul):not(ol)` looks strange to me.
It works for the first child of any selector inside the `os-problem-container` and `os-solution-container` that is not a list. Should it works for those in all of the books?

Reply:
That's actually connected with https://github.com/Connexions/cnx-recipes/issues/571 which will be solved by https://github.com/Connexions/webview/pull/1867
But it's also good to prevent this behaviour for lists in `os-eoc` and `os-eob`.